flixel/addons/tile/FlxTilemapExt.hx

```diff
@@ -735,6 +778,7 @@ class FlxTilemapExt extends FlxTilemap
 	function solveCollisionSlopeSoutheast(Slope:FlxObject, Object:FlxObject):Void
 	{
 		// Calculate the corner point of the object
+		if(Object.x<Slope.x-16) return;
```

Comment:
A hardcoded value of `16`? I'm guessing this only works with tilemaps that happen to use that as their tile size? ;)